docs/configuration/general/virus_scan.rst

```diff
@@ -0,0 +1,55 @@
+.. _configuration_general_virus_scan:
+
+Virus scan with ClamAV
+=======================
+
+In Open-Forms, it is possible to scan files uploaded by a user for possible viruses. The scan is done with `ClamAV`_.
+The scan happens as soon as the user uploads a file in the **File upload** component and if it is found to contain
```

Comment:
Maybe add a reference/backlink to this part in the documentation section that describes the upload/processing flow? https://open-forms.readthedocs.io/en/latest/developers/backend/file-uploads.html?highlight=File%20uploads#upload-flow

Reply:
sorry, I meant the other way around - have the developer documentation reference the virus scanning step. Now if you're reading the dev docs, you have no clue that virus scanning can happen.